Formula/gogs.rb

```diff
@@ -0,0 +1,41 @@
+class Gogs < Formula
+  desc "Go Git Service: A painless self-hosted Git service"
+  homepage "https://gogs.io"
+  url "https://cdn.gogs.io/gogs_v0.9.48_darwin_amd64.zip"
```

Comment:
No problem, I'll just cancel the PR. Building from source involves a lot of gymnastic with go and its build system, the binary install is *far* simpler.